src/libraries/CalldataDecoder.sol

```diff
@@ -50,4 +52,46 @@ library CalldataDecoder {
             }
         }
     }
+
+    /// @notice Decode the `_arg`-th element in `_bytes` as `bytes`
+    /// @dev Reverts if the length is not what the encoding is expecting
+    // @param _bytes The input bytes string to extract a bytes string from
+    // @param _arg The index of the argument to extract
+    function toSafeBytes(bytes calldata _bytes, uint256 _arg) internal pure returns (bytes calldata res) {
```

Comment:
nit but i would rename `toBytesSafe` or `safeToBytes` this name confused me as to what it was going to do